### ISSUE TITLE
iOS back button misalignment and status bar color not inherited

### ISSUE THREAD

**Issue by ThorConzales** [ionic-team/ionic#13046]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ x] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[x ] bug report
[ ] feature request

**Current behavior:**
Back button is misaligned in latest version and when I upgraded to X Code 9 (IIRC then this issue appeared after the XCode upgrade) then the status bar no longer inherits the color of the background and stays white. I even tried explicitly setting statusBar.overlaysWebView(true) to enable overlaying, but with no effect. I also tried upgrading to latest cordova-plugin-statusbar this also didn't help. Is it intended behavior as part of the iPhone X support? Attached a screenshot where default status bar is displayed also with the misaligned back button.

**Ionic info:**
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.12.0
    ionic (Ionic CLI) : 3.12.0

global packages:

    cordova (Cordova CLI) : 7.0.1 

local packages:

    @ionic/app-scripts : 3.0.0
    Cordova Platforms  : android 6.3.0 ios 4.5.1
    Ionic Framework    : ionic-angular 3.7.1

System:

    ios-deploy : 1.9.2 
    ios-sim    : 5.0.8 
    Node       : v6.11.3
    npm        : 3.10.10 
    OS         : macOS Sierra
    Xcode      : Xcode 9.0 Build version 9A235 

Misc:

    backend : pro
![iosbackbuttonbug](https://user-images.githubusercontent.com/9930079/31181462-0d8c7c0a-a92a-11e7-9ef1-d16d754341b6.png)

**Comment by ThorConzales** [comment]
Status bar issue is indeed separate issue relates to X Code 9, but the alignment issue seems to be Ionic related if it is intentional at all, since downgrading to 3.6.1 fixes it.

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

This issue becomes even more obvious with longer titles that have letters that dip below the line. In the following example, both are iPhone 6, left is iOS 10.3.1, right is iOS 11:

<img width="890" alt="screen shot 2017-10-05 at 10 41 57 am" src="https://user-images.githubusercontent.com/1526846/31237518-d084cc02-a9bc-11e7-8c42-66ee367a9bc2.png">

Tried installing latest nightly with iOS 11 fixes (`npm i ionic-angular@3.7.1-201710032000`) but that did not resolve, so still an issue.

Thank you for using Ionic

**Comment by ghenry22** [comment]
try installing the latest version of the statusbar plugin directly from the git repo rather than NPM. I had the same issue but it went away with the updated statusbar plugin on ios11

**Comment by mhartington** [comment]
Good the hear @ghenry22 .

Issue was resolved by installing the latest status bar plugin from git, not npm. 

Closing.

**Comment by ThorConzales** [comment]
I tried installing latest status bar plugin from git, but the issue persisted and only downgrading to 3.6.1 helped for me.

**Comment by mhartington** [comment]
I just pushed a fix for this. Will cut a nightly soon.

**Comment by mhartington** [comment]
Nightly is out


```
npm i ionic-angular@3.7.1-201710091909
```


Please install and verify this fixes things.

**Comment by ghenry22** [comment]
@ThorConzales - you're right I went back and tested with iOS 10 again with 3.7.1 and still saw the misalignment.

@mhartington - I have reproduced the problem with a 10.3.1 emulator and then installed nightly and tested on iOS 10.3.1 and 11 and the alignment of buttons in the header now looks correct.

As a side note I did notice on the iPhone X emulator that the footer gets more padding as well, when the keyboard comes into view if you slide the footer up above the keyboard it looks too big due to the extra padding that I presume is added for the safe area under the footer.

See screenshots, one taken from iphone 7plus emulator and one from iphone X emulator to see the footer padding difference.
<img width="394" alt="iphonex footer padding above keyboard too big" src="https://user-images.githubusercontent.com/7463213/31366127-0e7b3918-ada2-11e7-91cc-b5018c0263f0.png">
<img width="321" alt="iphone 7plus footer padding correct" src="https://user-images.githubusercontent.com/7463213/31366128-0ebe2d0e-ada2-11e7-958c-2a31ae72d3aa.png">
